openacademy/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+
+from . import models
```

Comment:
Please name files according to models: 
```suggestion
from . import openacademy_course
```

Reply:
done